semcore/notice/__tests__/index.test.jsx

```diff
@@ -95,6 +95,30 @@ describe('Notice', () => {
   });
 });
 
+describe('Notice.Title and Notice.Text', () => {
+  beforeEach(cleanup);
+
+  test.concurrent('Should correctly render with title and text', async ({ task }) => {
+    const component = (
+      <Notice>
+        <Notice.Label>
+          <div style={{ width: '16px', height: '16px', background: 'orange' }} />
+        </Notice.Label>
+        <Notice.Title>Your subscription has expired</Notice.Title>
+
+        <Notice.Content>
+          <Notice.Text>49 out of your 50 projects are now locked.</Notice.Text>
```

Comment:
I think, `Notice.Title` should be in `Notice.Content`

Reply:
moved